examples/rpl_udp/rpl.c

```diff
@@ -167,7 +167,7 @@ void rpl_udp_dodag(int argc, char **argv)
     printf("Part of Dodag:\n");
     printf("%s\n", ipv6_addr_to_str(addr_str, IPV6_MAX_ADDR_STR_LEN,
                                     (&mydodag->dodag_id)));
-    printf("my rank: %d\n", mydodag->my_rank);
+    printf("my rank: %d\n", NTOHS(mydodag->my_rank.u16));
```

Comment:
byteorder_ntohs(mydodag->my_rank)
No member access to .u16, .u32 etc should ever be needed.